CHANGELOG.md

```diff
@@ -3621,6 +3631,7 @@ Please read the changelog and test in your environment.
  - The initial versions  were rapildy iterated to deliver
    a working ingress controller.
 
+[3.2.3]: https://github.com/kong/kubernetes-ingress-controller/compare/v3.2.2...v3.2.3
```

Comment:
These links are unused in the last couple of versions because the headers (e.g. in lines 121 and 133) are not enclosed in `[]`. Can we add them in this PR?

Reply:
Ah, yeah, we could stick the diff links somewhere in the rendered content. Will do that separately.